### PR TITLE
Update binance api v3

### PR DESCRIPTION
**Pull request by halink0803**
- Update `quotePrecision` to `quoteAssetPrecision`
- Add api DELETE `api/v3/openOrders`

### CODE REVIEW

File: exchange/binance_format.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ type FilterLimit struct {
 type BinanceSymbol struct {
 	Symbol             string        `json:"symbol"`
 	BaseAssetPrecision int           `json:"baseAssetPrecision"`
-	QuotePrecision     int           `json:"quotePrecision"`
+	QuotePrecision     int           `json:"quoteAssetPrecision"`
```
Comment (secmask) [inline exchange/binance_format.go:70]: can we rename `QuotePrecision` to `QuoteAssetPrecision` too?

File: exchange/binance/binance_endpoint.go [hunk 2]
```diff
@@ -293,6 +293,26 @@ func (ep *Endpoint) CancelOrder(symbol string, id uint64) (exchange.Binacancel,
 	return result, err
 }
 
+// CancelAllOrders cancel all open order of an symbols
+func (ep *Endpoint) CancelAllOrders(symbol string) ([]exchange.Binaorder, error) {
+	var result []exchange.Binaorder
+	respBody, err := ep.GetResponse(
+		"DELETE",
```
Comment (secmask) [inline exchange/binance/binance_endpoint.go:300]: there is `http.MethodDelete` constant for method.

File: exchange/binance_format.go (hunk 1 above)
Comment (halink0803) [reply exchange/binance_format.go]: I have updated it.

File: exchange/binance/binance_endpoint.go (hunk 2 above)
Comment (halink0803) [reply exchange/binance/binance_endpoint.go]: I have changed it.

**Pull request merged**